actions/workflows/bwc_pkg_promote_all.yaml

```diff
@@ -54,9 +61,9 @@ tasks:
     next:
       - do:
           - promote_bwc_enterprise
-          - promote_st2_auth_ldap
-          - promote_st2flow
-          - promote_bwc_ui
+          # - promote_st2_auth_ldap
+          # - promote_st2flow
+          # - promote_bwc_ui
```

Comment:
Are we changing this to sequential execution now?

Reply:
Yes. I had an issue running them in parallel, and I wanted to finish up this PR since it's been open for quite some time. This will be an area of investigation in the future (or now, if you want this to be a gating issue).

Reply:
Ok. Please open an issue in this repo with errors and details for the parallel execution. Please remove the commented out codes here and below but add a comments in this task and explain why it was refactored to sequential execution.